models/model.js

```diff
@@ -31,3 +31,25 @@ exports.selectReviews = () => {
             return reviews;
         });
 };
+
+exports.selectCommentsByReviewId = (id) => {
+    return db
```

Comment:
This is one way of doing it. Have a think about going forward, we'll also need to check if things exist in the database for post requests as well. Could this be extracted out into a util function that checks if something exists in the database?